packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -365,6 +379,11 @@ abstract class ClusterBase extends Resource implements ICluster {
    */
   public abstract readonly connections: ec2.Connections;
 
+  /**
+   * Indicates if the parameter group can be updated.
+   */
+  protected abstract autoCreateParameterGroup: boolean;
```

Comment:
This is weird...I would prefer this to be managed through class inheritance, instead of through this magic boolean value that every implementation has to check. 

Or better yet, if this depends only on the parameter group, the parameter group itself should have this property.